docs/userguide.md

```diff
@@ -305,6 +305,27 @@ To cleanup the example simply delete the namespace:
 kubectl delete ns test-namespace
 ```
 
+### Troubleshooting
+
+If you found the federated resources were not propagated to the member clusters, you can always
```

Comment:
s/If you found the federated resources were not propagated/If federated resources are not propagated as expected/
s/always//